### PR TITLE
add introducing Core blog post

### PR DESCRIPTION
**Pull request by RasPhilCo**

### CODE REVIEW

File: website/blog/2021-02-12-introducing-oclif-core.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+`@oclif/core` ("Core") simplifies the development experience and introduces highly requested new features.
+
+Core combines the essential oclif packages into one "core" package, aptly named `@oclif/core`. 
+
+Core also introduces:
+- A default command option so "multi" CLIs can execute a default command at the bare bin
+- Use colon or spaced command syntax
+- Async parsing 
+- Command piping to arguments
+- Global flags
```
Comment (RasPhilCo) [inline website/blog/2021-02-12-introducing-oclif-core.md:22]:
```suggestion
```

File: website/blog/2021-02-12-introducing-oclif-core.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+We are excited to announce the next iteration of the oclif project today: `@oclif/core`.
+
+We have learned a lot in the last three years of developing oclif and developing on oclif, running millions of commands a day via Heroku & Salesforce CLIs.
+`@oclif/core` ("Core") simplifies the development experience and introduces highly requested new features.
+
+Core combines the essential oclif packages into one "core" package, aptly named `@oclif/core`. 
+
+Core also introduces:
+- A default command option so "multi" CLIs can execute a default command at the bare bin
```
Comment (RasPhilCo) [inline website/blog/2021-02-12-introducing-oclif-core.md:18]:
```suggestion
- A default command option
```

File: website/blog/2021-02-12-introducing-oclif-core.md [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+
+With default command functionality, Core simplifies the oclif project and removes the notion of single vs multi command CLIs. Core CLIs can have 1 or many commands.
+
+Along with Core, we have combined the oclif and oclif-dev CLIs into a single CLI with a new AWS S3 compatible publishing scheme.
+
+### What to expect in the near future
+
+Core is in pre-release beta and being actively developed for new internal Salesforce CLIs.
+
+Much documentation needs to be written in the comming months including migration paths. Look for forthcoming blog posts and documentation on [oclif.io](https://oclif.io).
```
Comment (lnoellemorgan) [inline website/blog/2021-02-12-introducing-oclif-core.md:31]: comming -> coming

File: website/blog/2021-02-12-introducing-oclif-core.md [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+I've written Core with current CLIs migration in mind. Migration onto it should be as painless as possible with many exports remaining entirely unchanged.
+
+Early this summer, tentively June 1, we will release Core v1. Core's release will coincide with major bumps to many oclif plugin packages. See the compatablility matrix below (which will be included in documentaion).
```
Comment (lnoellemorgan) [inline website/blog/2021-02-12-introducing-oclif-core.md:35]: compatablility -> compatibility
documentaion -> documentation

File: website/blog/2021-02-12-introducing-oclif-core.md [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+
+At Core's v1 release, the current essential oclif packages (namely: command, config, errors & parser) will go into maintenance mode until Jan 2022. They will receive _only_ bug and security fixes and they remain compatable with current versions of the oclif and oclif-dev CLIs.
```
Comment (lnoellemorgan) [inline website/blog/2021-02-12-introducing-oclif-core.md:37]: compatable -> compatible

File: website/blog/2021-02-12-introducing-oclif-core.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+
+Along with Core, we have combined the oclif and oclif-dev CLIs into a single CLI with a new AWS S3 compatible publishing scheme.
+
+### What to expect in the near future
+
+Core is in pre-release beta and being actively developed for new internal Salesforce CLIs.
+
+Much documentation needs to be written in the comming months including migration paths. Look for forthcoming blog posts and documentation on [oclif.io](https://oclif.io).
+
+I've written Core with current CLIs migration in mind. Migration onto it should be as painless as possible with many exports remaining entirely unchanged.
```
Comment (lnoellemorgan) [inline website/blog/2021-02-12-introducing-oclif-core.md:33]: You use "I" as the pronoun here but sign it "the oclif team". I would change this to "we" or change the signature to your name.

**Review by lnoellemorgan — CHANGES_REQUESTED** [review]

**Pull request merged**